django/contrib/admin/checks.py

```diff
@@ -465,8 +465,6 @@ def _check_ordering_item(self, obj, model, field_name, label):
         else:
             if field_name.startswith('-'):
                 field_name = field_name[1:]
-            if field_name == 'pk':
-                return []
```

Comment:
I think this is a change in behavior although there is no test failure and I don't think it makes any sense anyway.